### ISSUE TITLE
ArrowControl disappear when removing ToFadeDuration.

### ISSUE THREAD

**Issue by AhmedSalahKhodair** [AndreiMisiukevich/CardView#151]
When I either remove `ToFadeDuration` or setting it by 0, the arrow disappear and become invisible, but it is still clickable tho.

For trial purpose:
[here](https://github.com/AndreiMisiukevich/CardView/blob/87326b56e1a3690fa42145cefe9cddb171fe0d29/PanCardViewSample/PanCardViewSample/Views/CarouselSampleXamlView.xaml#L43): remove `ToFadeDuration="2500"`.

For removing ambiguity: I need it both visible and clickable

**Comment by AndreiMisiukevich** [comment]
@AhmedSalahKhodair hi, thanks, got it.

Does anybody want to debug, realize the reason, fix and make PR?
Must be something easy :)

**Comment by AndreiMisiukevich** [comment]
I think, adding ```Opacity = 1;``` here https://github.com/AndreiMisiukevich/CardView/blob/master/PanCardView/Controls/ArrowControl.cs#L144 will solve the issue

**Comment by AhmedSalahKhodair** [comment]
It works but not stably.
Consider the left arrow case:
When I set `IsCyclical="False"` and remove `SlideShowDuration="3500"` and swipe to the first page (the left arrow disappear and that's good) but when I swiped to the second page (the left arrow disappear too and that's wrong) and when I swiped to next it appears again and works fine, and if I repeat the swiping process again, error happens again.
I will spend more time on it after few hours.
